model/src/main/kotlin/utils/SortedSetConverters.kt

```diff
@@ -93,4 +77,22 @@ class SnippetFindingSortedSetConverter : StdConverter<Set<SnippetFinding>, Sorte
         value.toSortedSet(compareBy<SnippetFinding> { it.sourceLocation.path }.thenByDescending { it.snippet.purl })
 }
 
+private fun Provenance.sortKey(): String =
```

Comment:
The "sort" verb as part of the function name sounds as if the function itself would already sort. Maybe name this "toKey()", "getSortKey()", "getSelector()" or similar?